README.md

```
@@ -136,9 +136,15 @@ Use subcommand `fanyi config set <key> <value>` to set configuration options.
 Example:
 
 ```bash
-$ fanyi config list                       // list all configuration options
-$ fanyi config set iciba false            // disable iciba globally
-$ fanyi config set deepseek false             // disable deepseek globally
-$ fanyi config set color false            // disable color globally
-$ fanyi config set LLM_API_KEY your-api-key // set LLM_API_KEY
+$ fanyi config list                                   // list all configuration options
```

markdownlint-cli2 (0.17.2)

Dollar signs used before commands without showing output
null

(MD014, commands-show-output)